.github/workflows/handler-test.yml

```diff
@@ -43,6 +43,13 @@ jobs:
           AWS_DEFAULT_REGION: us-west-2
         run: terraform validate -no-color
 
+      - name: Terraform Apply
+        id: apply
+        working-directory: ./tests/public-install
+        env:
+          AWS_DEFAULT_REGION: us-east-2
```

Comment:
This shouldn't be required since the region is defined within the TFC workspace. I had to add this to the validation step because it runs locally and not having it defined triggered an error.